drf_dx_datagrid/filters.py

```diff
@@ -23,38 +23,48 @@ def __to_django_operator(self, operator):
         elif operator == "<=":
             return "__lte"
         else:
-            return "__" + operator
+            return "__" + operator if is_case_sensitive else "__i" + operator
```

Comment:
This condition is required:
operator in ["exact", "contains","startswith","endswith", "regex"]

Reply:
Shure, but in case of case insensitive behavior they schould be:
["iexact", "icontains","istartswith","iendswith", "iregex"]